engines/bladerunner/ui/vk.cpp

```diff
@@ -456,9 +463,9 @@ void VK::init() {
 	_buttons->defineImage(2, Common::Rect(515, 368, 538, 398), nullptr,    _shapes[6],  _shapes[7],  nullptr);
 	_buttons->defineImage(3, Common::Rect(548, 368, 571, 398), nullptr,    _shapes[8],  _shapes[9],  nullptr);
 	_buttons->defineImage(4, Common::Rect(581, 368, 604, 398), nullptr,    _shapes[10], _shapes[11], nullptr);
-	_buttons->defineImage(5, Common::Rect( 31, 363,  65, 392), nullptr,    _shapes[0],  _shapes[1], _vm->_textVK->getText(0));
```

Comment:
This was done on purpose, for the horizontal alignment.

Reply:
As noted, this is a side-effect of me running AStyle without checking for its side-effects.